tests/formats/mmtf.cpp

```diff
@@ -176,6 +176,17 @@ TEST_CASE("Read files in MMTF format") {
         CHECK(frame[*edo.begin()].get("altloc")->as_string() == "B");
     }
 
+    SECTION("Check symmetry") {
+    #ifndef CHEMFILES_WINDOWS
+        if (!is_valgrind_and_travis()) {
+            auto file = Trajectory("data/mmtf/5IRE.mmtf");
+
+            auto frame = file.read();
+            CHECK(frame.size() == 661440);
```

Comment:
It would be nice to have a smaller file to check symmetry operations. If there is none on the PDB archive, how about an "artificial" file, manually created?

Reply:
I purposefully chose an entry with a large number of symmetry operations, but I can replace the test with an entry with a smaller number.

Reply:
Turns out 5A1I is also an entry which reuires symmetry operations, so I just exanded the test. I'll merge this tomorrow unless there's a major objection.